### ISSUE TITLE
Prior restraint length data

### ISSUE THREAD

**Issue by chigby** [freedomofpress/pressfreedomtracker.us#1784]
Originally part of https://github.com/freedomofpress/pressfreedomtracker.us/issues/1478

For the Prior Restraint category:
> Add date entry fields for date ordered and date lifted
>
> If possible, we would love for this to essentially be used to field a "length of restraint" display metadata / potentially make "length of restraint" searchable. Date itself does not need to be added to filter.

**Comment by harrislapiroff** [comment]
Should check with PFT if this is still desired @KAMcCudden @stephaniesugars

**Comment by stephaniesugars** [comment]
Yes, definitely still interested. And similar functionality could potentially also be applied to the Arrest category, for duration of detention.
